src/components/EditorsColumn.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import isEmpty from 'lodash/isEmpty';
+import includes from 'lodash/includes';
+import partial from 'lodash/partial';
+
+import EditorContainer from './EditorContainer';
+import Editor from './Editor';
+
+class EditorsColumn extends React.Component {
+  _allErrorsFor(language) {
+    if (language === 'javascript') {
+      return this.props.errors.javascript.items.
+      concat(this.props.runtimeErrors);
+    }
+
+    return this.props.errors[language].items;
+  }
+
+  render() {
```

Comment:
My opinions aren't super strong, but they boil down to:
- If I'm using Flow, use the `class` version because the speccing of `props` looks a little nicer than with stateless IMHO
- Otherwise, use stateless if there are no non-static methods (other than `render`). As an example, `_allErrorsFor` currently depends on `this.props`, so that would tilt me towards using `class`.

Of course `_allErrorsFor` could be refactored to be `this`-independent, which is what I'd do if you want to go stateless.